src/components/fields/location-field/location-modal/location-search/location-search.styles.ts

```diff
@@ -42,23 +42,23 @@ export const SearchBarContainer = styled.div<{ hasScrolled?: boolean }>`
 	}
 `;
 
-export const SearchBarIconButton = styled.button`
+export const SearchBarIconButton = styled.button<{ disableCursor?: boolean }>`
```

Comment:
instead of passing prop, you can style via the `&:disabled` selector

Reply:
ahh ok, updated, thanks!